shared/AppInsightsCore/Tests/Unit/src/CookieManager.Tests.ts

```diff
@@ -608,7 +646,10 @@ export class CookieManagerTests extends AITestClass {
             test: () => {
 
                 let core = new AppInsightsCore();
-                core.initialize({
+                let cookieValue = this._testCookies[newKey];
```

Comment:
WTF!
You have completely removed the core.initialize call and in the process introduced a bunch of syntax errors.

revert the last commit and fix correctly putting the assertions in the correct location